overlay/master_server.py

```diff
@@ -529,6 +567,8 @@ def SubmitValidationJob(self, request: ValidationJobRequest, context):
         else:  # Default or static budget
             job_id, worker_responses = self.process_job_with_normal_budget(request)
 
+        # Flattened list of metrics, instead of being nested in respective Worker responses
+        flattened_metrics: list = []
```

Comment:
So now `SubmitValidationJob` returns a response with all the metrics flattened, versus previously where we had the metrics underneath worker responses for experiment benchmarks.
**Old**:
{
  <top_level_stuff>,
  worker_responses: [
    {worker_response_1, metrics: [ {metric_0}, ... ],
    {worker_response_2, metrics: [ {metric_0}, ... ],
    ...
    {worker_response_K, metrics: [ {metric_0}, ... ]
  ]
}
```


**New**:
```
{
  <top_level_stuff>,
  metrics: [
    {<validation_metric_1},
    {<validation_metric_2},
    ...,
    {<validation_metric_N}
  ]
}
```